### DISCUSSION TITLE
Time greater or lower than

### DISCUSSION THREAD

**Discussion by ggukggukee** [prisma/prisma#18151] [Q&A] [answered]
Hello!

If I have column `Time` with type `@db.Time(x)`. Then how can I search time between range? (PostgreSQL)

```js
const groups = await prisma.group.findMany({
   where: {
      time: {
         lte: '11:20',
         gte: '12:34',
      },
   },
});
```

**Comment by ludralph** [comment, marked as answer]
Hi @ggukggukee 👋 

you can do something like this 

```js
let date = new Date()
  date.setHours(11)
  date.setMinutes(20)
  date.setSeconds(0)

let date2 = new Date()
date.setHours(12)
date.setMinutes(34)
date.setSeconds(0)


constgroups = await prisma.group.findMany({
  where: {
    time: {
      gte: date,
      lt:  date2
    },
  },
});
```
Do let me know if this works for you.

**Reply by nikolasburk** [reply]
Hey @ggukggukee 👋 

do you have any more questions around this issue? If no, it would be really helpful if you could quickly **mark this Discussion as _answered_** to indicate that your issue has been resolved. 

In case you do have any outstanding questions, feel free to let us know and we'd be happy to help!
